assets/src/scripts/monitoring-location/components/hydrograph/graph-brush.js

```diff
@@ -62,23 +83,38 @@ export const drawGraphBrush = function(container, store) {
         })
         .call(link(store, (svg, {layout, hydrographBrushOffset, xScale}) => {
             let selection;
+            layoutHeight = layout.height;
 
             const graphBrush = brushX()
                 .on('brush end', brushed);
 
             svg.select('.brush').remove();
 
             const group = svg.append('g').attr('class', 'brush')
-                .attr('transform', `translate(${layout.margin.left},${layout.margin.top})`);
+                .attr('transform', `translate(${layout.margin.left}, ${layout.margin.top})`);
 
+            graphBrush.handleSize([1]); // make default handle 1px wide
             graphBrush.extent([[0, 0], [layout.width - layout.margin.right, layout.height - layout.margin.bottom - layout.margin.top]]);
 
+            const brushResizePath = function(d) {
```

Comment:
A short comment here would be good to explain what this does.

Reply:
I'll do my best.